Update desription for ocdev create

cdrage opened the pull request:
Updates the description for ocdev create with a bit more information.

kadel commented on cmd/create.go:
```diff
@@ -20,18 +20,24 @@ var (
+
+If component name is not provided, component type value will be used for the name.
+
+OpenShift built-in types: httpd, nodejs, perl, php, python, ruby, wildfly
```
I'm not sure if it is a good idea to have the list here. It will be hard to keep this update.
Just referring to `ocdev component list` is I think enough.

cdrage commented:
Thanks @kadel I've updated it and added one more line with regards to using Python, but that's it. You're right, since OpenShift is constantly adding new containers / build images, it'd be hard to maintain.

kadel commented on cmd/create.go:
```diff
@@ -20,18 +20,25 @@ var (
+  # Create a Ruby component
+  ocdev create ruby
+	
+	# Create a Python component
```
indentation. There should be two spaces at the beginning to alight it with everything else.

cdrage replied:
stupid vim haha

cdrage replied:
woops vim

cdrage commented:
Thx! Rebased :+1:
